dataflow-pipelines/src/main/kotlin/org/ostelco/dataflow/pipelines/DeployPipeline.kt

```diff
@@ -0,0 +1,63 @@
+package org.ostelco.dataflow.pipelines
+
+import ch.qos.logback.classic.util.ContextInitializer
+import org.apache.beam.runners.dataflow.DataflowRunner
+import org.apache.beam.runners.dataflow.options.DataflowPipelineOptions
+import org.apache.beam.sdk.Pipeline
+import org.apache.beam.sdk.options.PipelineOptions
+import org.apache.beam.sdk.options.PipelineOptionsFactory
+import org.ostelco.dataflow.pipelines.definitions.DataConsumptionPipelineDefinition
+import org.ostelco.dataflow.pipelines.definitions.DummyPipelineDefinition
+import org.ostelco.dataflow.pipelines.definitions.PipelineDefinition
+
+enum class PipelineDefinitionRegistry(val pipelineDefinition: PipelineDefinition) {
+    DATA_CONSUMPTION(DataConsumptionPipelineDefinition),
+    DUMMY(DummyPipelineDefinition),
+}
+
+fun main(args: Array<String>) {
+    System.setProperty(ContextInitializer.CONFIG_FILE_PROPERTY, "config/logback.xml")
+    DeployPipeline().deploy(pipelineName = "DATA_CONSUMPTION")
+}
+
```

Comment:
Near the top of this class, because it is an important one,  there should be a descriptive text describing at least the intent behind the class and possibly (more important in other cases) some examples of happy day use cases.  For this class it may be of interest to list the command line parameters that are overridden (that would make the dead code below less necessary to understand the usage of the class).